contribs/gmf/examples/data/themes.json

```diff
@@ -0,0 +1 @@
te_field": "display_name", "disclaimer": "\u00a9 les contributeurs d\u2019OpenStreetMap", "legend": "true", "legend_rule": "Informations"}}], "id": 35, "name": "Enseignement"}], "metadata": {}}, {"name": "Patrimoine", "functionalities": {}, "id": 4, "icon": "http://geomapfish-demo.camptocamp.net/bge/wsgi/project/8143d12d42d5431baf576b39d32f18e5/img/patrimoine.jpeg", "children": [{"metadata": {}, "children": [{"layers": "bank", "name": "bank", "queryable": 1, "childLayers": [], "type": "internal WMS", "id": 100, "imageType": "image/jpeg", "metadata": {"is_checked": "true", "identifier_attribute_field": "display_name", "disclaimer": "\u00a9 les contributeurs d\u2019OpenStreetMap", "legend": "true", "legend_rule": "Banques"}}, {"layers": "place_of_worship", "name": "place_of_worship", "queryable": 1, "childLayers": [], "type": "internal WMS", "id": 104, "imageType": "image/jpeg", "metadata": {"is_checked": "true", "identifier_attribute_field": "display_name", "disclaimer": "\u00a9 les contributeurs d\u2019OpenStreetMap", "legend": "true", "legend_rule": "Lieux de culte"}}], "id": 7, "name": "Patrimoine"}], "metadata": {}}, {"name": "Gestion des eaux", "functionalities": {}, "id": 3, "icon": "http://geomapfish-demo.camptocamp.net/bge/wsgi/project/8143d12d42d5431baf576b39d32f18e5/img/gestion_eaux.jpeg", "children": [{"metadata": {}, "children": [{"layers": "zoo", "name": "zoo", "queryable": 1, "childLayers": [], "type": "internal WMS", "id": 108, "imageType": "image/jpeg", "metadata": {"is_checked": "true", "identifier_attribute_field": "display_name", "disclaimer": "\u00a9 les contributeurs d\u2019OpenStreetMap", "legend": "true", "legend_rule": "Zoos"}}, {"layers": "fuel", "name": "fuel", "queryable": 1, "childLayers": [], "type": "internal WMS", "id": 124, "imageType": "image/jpeg", "metadata": {"is_checked": "true", "identifier_attribute_field": "display_name", "disclaimer": "\u00a9 les contributeurs d\u2019OpenStreetMap", "legend": "true", "legend_rule": "Stations service"}}], "id": 8, "name": "Gestion des eaux"}], "metadata": {}}, {"name": "Paysage", "functionalities": {}, "id": 2, "icon": "http://geomapfish-demo.camptocamp.net/bge/wsgi/project/8143d12d42d5431baf576b39d32f18e5/img/paysage.jpeg", "children": [{"metadata": {}, "children": [{"layers": "alpine_hut", "name": "alpine_hut", "queryable": 1, "childLayers": [], "type": "internal WMS", "id": 123, "imageType": "image/jpeg", "metadata": {"is_checked": "true", "identifier_attribute_field": "display_name", "disclaimer": "\u00a9 les contributeurs d\u2019OpenStreetMap", "legend": "true", "legend_rule": "Cabanes alpines"}}, {"layers": "zoo", "name": "zoo", "queryable": 1, "childLayers": [], "type": "internal WMS", "id": 108, "imageType": "image/jpeg", "metadata": {"is_checked": "true", "identifier_attribute_field": "display_name", "disclaimer": "\u00a9 les contributeurs d\u2019OpenStreetMap", "legend": "true", "legend_rule": "Zoos"}}], "id": 9, "name": "Paysage"}], "metadata": {}}], "errors": []}
```

Comment:
You probably take this from the gmf-layertree PR.
I've updated it because in this json, the concerned instance is "bge" (must be 1.6 or 2.0...).
Now I use '1.6'.

Perhaps we need another stable instance to guarantee that the example works in the future...